includes/admin.php

```
@@ -484,8 +591,26 @@ function add_action_links( $actions ) {
 /**
- * Display Warning that Safety Net is activated.
+ * Pause WooCommerce Subscriptions renewal and failed payment retry scheduled actions
  *
+ * @return void
+ */
+function pause_renewal_actions() {
+	if ( 'on' === get_option( 'safety_net_pause_renewal_actions_toggle' ) ) {
+		require_once __DIR__ . '/classes/class-actionscheduler-custom-dbstore.php';
+		add_filter(
+			'action_scheduler_store_class',
+			function ( $class ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$class'. (undefined)

(UnusedFormalParameter)